graylog2-server/src/test/java/org/graylog2/utilities/date/NaturalDateParserTest.java

```diff
@@ -55,15 +54,54 @@ public void testParseFailsOnEmptyDate() throws Exception {
     }
 
     @Test
-    @Ignore
-    public void testTemporalOrder() throws Exception {
+    public void testDefaultTZ() throws Exception {
         NaturalDateParser p = new NaturalDateParser();
 
+        NaturalDateParser.Result today = p.parse("today");
+        assertThat(today.getFrom()).as("From should not be null").isNotNull();
+        assertThat(today.getTo()).as("To should not be null").isNotNull();
+        assertThat(today.getDateTimeZone().getID()).as("should have the Etc/UTC as Timezone").isEqualTo("Etc/UTC");
```

Comment:
We should also ensure that from/to are what we expect for the `today` keyword, especially for the following test cases where timezones are being used.

Reply:
This is the PR that does not align to day start/end. So in this case, we would either test something that's outdated in the near future or we would test the underlying lib (natty) for correctly handling parsing today with different timezones. Testing natty in this regard might be out of scope here.

Reply:
IMO natty is an implementation detail here, so we should test what our interface (`NaturalDateParser`) is supposed to do and that is returning correct boundaries for a given keyword.